crates/worker/src/purgatory.rs

```diff
@@ -116,6 +111,63 @@ impl Inner {
             .push((id, to_purge_timestampe));
     }
 
+    async fn global_state_purge(db_client: DynamoDBClient, s3_client: S3Client) {
+        const SYNC_FROM_OFFSET: Option<Duration> = PURGE_INTERVAL.checked_mul(6);
```

Comment:
We can use just simple variable here, not const

Reply:
Variable isn't guaranteed to be optimized to be in readonly part of binary. SYNC_FROM_OFFSET is evaluated at compile time as stated [here](https://doc.rust-lang.org/reference/const_eval.html#const-functions), so I would propose to leave it like that